app/views/lists/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= render 'shared/errors', model_instance: @list %>
+
+<%= form_with(model: [ @board, @list ], local:true) do |f| %>
+    <div class="form-group">
+        <%= f.label :name %>
+        <%= f.text_field :name, class: "form-control", placeholder: "Name" %>
+    </div>
+    <%= f.hidden_field :board_id %>
+    <%= hidden_field_tag :board_id %>
```

Comment:
Heh strzelasz

Reply:
nie strzelam, jak inaczej to zrobić?

Reply:
No nie wiem po co Ci ten board id w hidden jak on bedzie w routes. Przeciez masz nested resources